Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.0.12"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```

Comment:
What are you using from this package?

Reply:
SortedDict in the examples/alex_sandboxl (not included in push. I can remove that if you prefer)

Reply:
Thanks.  You can add that to the Project.toml file for your example/sandbox code.